src/channeloutput/UDPOutput.cpp

```diff
@@ -254,6 +255,10 @@ int UDPOutput::Init(Json::Value config) {
                 //DDP types
                 outputs.push_back(new DDPOutputData(s));
                 break;
+            case 6:
```

Comment:
I wonder if there is a reason the other output types have two "types" Maybe ZCPP should too?

Reply:
My guess is this is why there are two type constants for DDP...
![image](https://user-images.githubusercontent.com/57245161/100455903-1ec5df80-3085-11eb-8742-5359a051f9dd.png)